### PR TITLE
Cle/6038 send to stlt

### PR DESCRIPTION
**Pull request by clediggins-usds**
This PR ensures that a universal pipeline messages makes it all the way through to the 'send' step with the correct task, action, and report_file records.

Test Steps:
1. Run unit tests
OR
1. Send a universal pipeline message
2. Check DB to see that there are steps for receive, convert, route, translate, batch, send

## Changes
- add new 'convert' TaskAction
- update next step on receive to be 'convert' when it is universal pipeline
- update batch function to work for both covid and UP
- update batch function to make it testable with injectable mocked components
- nullify next_action on task after it happens for UP, to match covid pipeline activities
- insert tasks and actions for each step
- create hl7MessageRSHelpers to take an HL7 message, create a RS Report based off of it, and add proper lineage
- add batch function tests for UP. Note: There were not tests for covid batch and still are not. Tech debt.

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] Added tests?

### CODE REVIEW

**Comment by carlosfelix2** [comment]
I get a failure that looks to be on the send side:
```
09:13:00.797 [pool-2-thread-48][72] WARN  gov.cdc.prime.router.azure.ActionHistory - ***** FAILURE: sanity check comparing old headers list to new ReportFiles list:
[2022-10-12T13:13:00.798Z] header.receiverName = co-phd.full-elr-hl7, but reportFile has null.null
```

**Comment by carlosfelix2** [comment]
The files in the ready blob folder now have the schema name of CO.  Cool!

File: prime-router/src/main/kotlin/azure/Event.kt [hunk 1]
```diff
@@ -41,6 +42,7 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
                 PROCESS_WARNING -> TaskAction.process_warning
                 PROCESS_ERROR -> TaskAction.process_error
                 RECEIVE -> TaskAction.receive
+                CONVERT -> TaskAction.convert
```
[the remark this answers is not in the document]
Comment (clediggins-usds) [reply prime-router/src/main/kotlin/azure/Event.kt:45]: Convert = 'turn hl7 into fhir'
Translate = 'make the fhir into output (could be STLT specific) hl7'

File: prime-router/src/main/kotlin/fhirengine/utils/HL7MessageRSHelpers.kt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import gov.cdc.prime.router.ReportId
+import gov.cdc.prime.router.Source
+import gov.cdc.prime.router.azure.ActionHistory
+import gov.cdc.prime.router.azure.BlobAccess
+import gov.cdc.prime.router.azure.Event
+import gov.cdc.prime.router.azure.ProcessEvent
+import gov.cdc.prime.router.azure.ReportEvent
+import gov.cdc.prime.router.azure.db.tables.pojos.ItemLineage
+
+object HL7MessageRSHelpers {
```
[the remark this answers is not in the document]
Comment (clediggins-usds) [reply prime-router/src/main/kotlin/fhirengine/utils/HL7MessageRSHelpers.kt:16]: Updated, no specific strong feeling about this.

**Review by carlosfelix2 — APPROVED** [review]

**Review by TurkeyFried — APPROVED** [review]
Nice! I will help one of my PRs too

**Pull request merged**